content/applications/hr/employees/new_employee.rst

```diff
@@ -66,8 +49,8 @@ Optional fields
 
 .. tip::
    To make edits to the selected :guilabel:`Department`, :guilabel:`Manager`, :guilabel:`Coach`, or
-   :guilabel:`Company`, click the :guilabel:`External Link` button next to the respective selection.
-   The :guilabel:`External Link` button opens the selected form, allowing for modifications. Click
+   :guilabel:`Company`, click the :guilabel:`Internal Link` button next to the respective selection.
+   The :guilabel:`Internal Link` button opens the selected form, allowing for modifications. Click
    :guilabel:`Save` after any edits are made.
```

Comment:
In the sentence, it says "to make edits to the selected..." - because the internal link icon only appears after a selection is made. You won't see it when the drop-down is deployed, only after it is selected. Since I give the instruction to make the selection right above- it seems clear.